README.md

```diff
@@ -95,7 +97,9 @@ The `CommonFactory` reads a RabbitMQ configuration from the rabbitMQ.json file.
   "minConnectionRecoveryTimeout": 10000,
   "maxConnectionRecoveryTimeout": 60000,
   "prefetchCount": 10,
-  "messageRecursionLimit": 100
+  "messageRecursionLimit": 100,
+  "secondsToCheckVirtualQueueLimit": 10,
+  "batchesToCheckVirtualQueueLimit": 10000
```

Comment:
Redundant, this value should depend on the virtual limit value

Reply:
removed